user_modules/IsReady_Template/ATF_Interface_IsReady_missing_SingleRPC_Template.lua

```diff
@@ -376,20 +376,26 @@ config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd40
 										EXPECT_NOTIFICATION("OnHashChange")
 										:Times(0)
 									end
-								
+
 								else
 									if(mob_request.name == "DeleteCommand" or mob_request.name == "DeleteSubMenu") then
 										-- According to APPLINK-27079; APPLNIK-19401
 										--mobile side: expect RPC response
 										EXPECT_RESPONSE(cid, {success = false, resultCode = "INVALID_ID"})
-										
+
 									elseif(mob_request.name == "UnsubscribeVehicleData" or (mob_request.name == "UnsubscribeWayPoints")) then
 										-- According to APPLINK-27872 and APPLINK-20043
 										-- mobile side: expect RPC response
 										EXPECT_RESPONSE(cid, {success = false, resultCode = "IGNORED"})
 									else
 										if (TestData[i].resultCode == "") then
-											EXPECT_RESPONSE(cid, { success = TestData[i].success , resultCode = TestData[i].expected_resultCode})
+											if mob_request.name == "Slider"  then
+												-- because Slider has timeout parameter
+												local timeout = mob_request.params.timeout +11000

```

Comment:
@VVeremjova Please add indent after "+"
local timeout = mob_request.params.timeout +` `11000

Also I propose replace "magic number" to variable.